### PR TITLE
Header, footer and clear pages

### PR DESCRIPTION
**Pull request by BlazeOfSoul**

### CODE REVIEW

File: client/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+    path:'SignUp',
+    component: SignupFormComponent
+  },
+  {
+    path:'Teacher',
+    component: TeachersComponent
+  },
+  {
+    path:'ActiveDay',
+    component: ActiveDayComponent
```
Comment (bogomazdmitry) [inline client/src/app/app-routing.module.ts:28]: What is it?

File: client/src/app/components/layout/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<footer class="bg-light text-center" id="footer">
+  <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2)">
```
Comment (bogomazdmitry) [inline client/src/app/components/layout/footer/footer.component.html:2]: It is bad decision. Better to create css variables for colors

File: client/src/app/components/layout/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<mat-toolbar>
+  <a routerLink="" class="nav-link mx-2">Raf Rate</a>
+  
+  <a routerLink="ActiveDay" class="nav-link mx-5">Мероприятия</a>
+  <a routerLink="Teacher" class="nav-link">Преподаватели</a>
+
+  <span class="example-spacer"></span>
+
+  <button routerLink="SignIn" class="btn btn-light my-2 my-sm-0 mx-2">Войти</button>
```
Comment (bogomazdmitry) [inline client/src/app/components/layout/header/header.component.html:9]: Better to move margins to css file

**Review by bogomazdmitry — CHANGES_REQUESTED** [review]

File: client/src/app/app-routing.module.ts (hunk 1 above)
Comment (BlazeOfSoul) [reply client/src/app/app-routing.module.ts:28]: It's path and component wich will be use when we route to ActiveDay page

Comment (bogomazdmitry) [reply client/src/app/app-routing.module.ts:28]: What is ActiveDay?

Comment (BlazeOfSoul) [reply client/src/app/app-routing.module.ts:28]: ActiveDay it's a new name for Event, we changed it on back end side. Entity had name Event, but you sayd that it bad name and we changed it to ActiveDay

**Comment by bogomazdmitry** [comment]
Scss variables please

**Pull request merged**